frontend/src/metabase/visualizations/visualizations/CartesianChart/use-chart-events.ts

```diff
@@ -322,16 +323,12 @@ export const useChartEvents = (
         settings,
       };
 
-      if (
-        !areMultipleCards &&
-        hasBreakout &&
-        visualizationIsClickable(clickData)
-      ) {
+      if (hasBreakout && visualizationIsClickable(clickData)) {
```

Comment:
Removing unnecessary boolean in conditional:

Since the `areMultipleCards` check is already occurring on line 304/305, we know that one we reach this check `!areMultipleCards` is true, since we passed the conditional above without exiting the function.